en/developer/tutorials/articles/03-upgrading-code-to-liferay-7.2/01-intro.markdown

```diff
@@ -57,6 +57,10 @@ Here are the code upgrade steps:
 
             Download the @product@ bundle you're upgrading to.
 
+        3.  [Migrate .cfg to .config](/docs/7-2/tutorials/-/knowledge_base/t/02-upgrading-your-development-environment.html#upgrade-cfg-to-config){.title commandId=upgrade-cfg-to-config}
```

Comment:
hey @simonjhy  these numbers don't line up in order. 

Also, we should put this entry in the same place as the properties upgrade.

Reply:
If we don't have a entry for upgrading properties (blade updateProps) we need to add one and move this entry near it since they are relatd.